librz/core/cmd/cmd_analysis.c

```diff
@@ -7998,3 +7938,43 @@ RZ_IPI RzCmdStatus rz_analyze_value_to_maps_handler(RzCore *core, int argc, cons
 	rz_core_analysis_value_pointers(core, state->mode);
 	return RZ_CMD_STATUS_OK;
 }
+
+RZ_IPI RzCmdStatus rz_analysis_info_show_handler(RzCore *core, int argc, const char **argv, RzCmdStateOutput *state) {
+	switch (state->mode) {
+	case RZ_OUTPUT_MODE_STANDARD:
+	case RZ_OUTPUT_MODE_JSON:
+		cmd_address_info(core, argv[1], state->d.pj);
+		break;
+	default:
+		rz_warn_if_reached();
+		break;
+	}
+	return RZ_CMD_STATUS_OK;
+}
+
+RZ_IPI RzCmdStatus rz_global_imports_handler(RzCore *core, int argc, const char **argv, RzCmdStateOutput *state) {
+	switch (state->mode) {
+	case RZ_OUTPUT_MODE_STANDARD:
+		if (RZ_STR_ISEMPTY(argv[1])) {
+			if (core->analysis->imports) {
+				char *imp;
+				RzListIter *iter;
+				rz_list_foreach (core->analysis->imports, iter, imp) {
+					rz_cons_printf("%s\n", imp);
+				}
```

Comment:
put the foreach outside the `switch`, so it's easier to add newer modes afterward without having to rewrite most of the function.